<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="wykresSinus" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleSinus">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop/Start</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="wykresProstokat" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="wykresPila" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleProstokat">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop/Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togglePila">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop/Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ustawienia</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
